components/camera/server_test.go

```diff
@@ -45,15 +45,17 @@ func TestServer(t *testing.T) {
 	cameraServer, injectCamera, injectCameraDepth, injectCamera2, err := newServer()
 	test.That(t, err, test.ShouldBeNil)
 
-	img := image.NewNRGBA(image.Rect(0, 0, 4, 4))
+	img := image.NewRGBA(image.Rect(0, 0, 4, 4))
 	var imgBuf bytes.Buffer
 	test.That(t, png.Encode(&imgBuf, img), test.ShouldBeNil)
 	var imgBufJpeg bytes.Buffer
-	test.That(t, jpeg.Encode(&imgBufJpeg, img, nil), test.ShouldBeNil)
+
+	test.That(t, libjpeg.Encode(&imgBufJpeg, img, &libjpeg.EncoderOptions{Quality: 75}), test.ShouldBeNil)
```

Comment:
We should standardize our compression quality to be the same (looks like 90) everywhere. I think you can decode this with the same options if you need to make the test still pass

Reply:
I agree, we can refactor to common encode/decode